source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -0,0 +1,239 @@
+New for 2021
+============
+
+A number of improvements have been made to FRC Control System software for 2021. This article will describe and provide a brief overview of the new changes and features as well as a more complete changelog for C++/Java WPILib changes. This document only includes the most relevant changes for end users, the full list of changes can be viewed on the various `WPILib <https://github.com/wpilibsuite/>`__ GitHub repositories.
+
+Major Features (All Languages)
+------------------------------
+
+- A WebSocket interface has been added to allow remote access to the robot simulator
+
+Major Features (Java/C++)
+-------------------------
+
+- **Holonomic Drive Controller**
+  - A controller that teams with holonomic drivetrains (i.e. swerve and mecanum) can use to follow trajectories. This also supports custom ``Rotation2d`` heading inputs that are separate from the trajectory because heading dynamics are decoupled from translational movement in holonomic drivetrains.
+
+WPILib
+------
+
+New Command-Based Library
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Watchdog and epoch reporting has been added to the command scheduler. This will let teams know exactly which command or subsystem is responsible for a loop overrun if one occurs.
+
+- A withName() command decorator has been added for Java teams. This lets teams set the name of a particular command using the `decorator pattern <docs/software/commandbased/convenience-features:Command Decorator Features>`.
+
+- Added a ``NetworkButton`` class, allowing users to use a boolean ``NetworkTableEntry`` as a button to trigger commands.
+
+- Added a ``simulationPeriodic()`` method to Subsystem. This method runs periodically during simulation, in addition to the regular ``periodic()`` method.
+
+Scheduling Functions
+^^^^^^^^^^^^^^^^^^^^
+
+- Added support for scheduling functions more often than the robot loop via ``addPeriodic()`` in TimedRobot. Previously, teams had to make a Notifier to run feedback controllers more often than the TimedRobot loop period of 20ms (running TimedRobot more often than this is not advised). Now, users can run feedback controllers more often than the main robot loop, but synchronously with the TimedRobot periodic functions so there aren't any thread safety issues.
```

Comment:
Now that the article on this has been merged, we can link to that instead of elaborating here with examples.